tests/ui/blacklisted_name.rs

```diff
@@ -43,3 +43,19 @@ fn issue_1647_ref_mut() {
     let ref mut baz = 0;
     if let Some(ref mut quux) = Some(42) {}
 }
+
+mod tests {
+    fn issue_7305() {
+        // `blackisted_name` lint should not be triggered inside of the test code.
+        let foo = 0;
+
+        // However, we an additional problematic code here that triggers another lint
+        // just to ensure that this code was compiled & processed.
+        let nOn_SnakeCase_ident = 10u8;
```

Comment:
I don't think this is necessary anymore and would like to avoid triggering unrelated lints in this test file.